js/ui/collection/ui.collection_widget.base.js

```diff
@@ -282,6 +282,10 @@ const CollectionWidget = Widget.inherit({
         this._updateFocusedItemState($target, false);
     },
 
+    _findActiveTarget($el) {
```

Comment:
It's not a good practice to shorten a word
https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers
```suggestion
    _findActiveTarget($element) {
```

Reply:
Okay, let's create our own style guide) I fixed it in this commit [25bf9f775185867becc632f8384161cc5d472868].